pkg/reconciler/route/reconcile_resources_test.go

```diff
@@ -165,12 +165,13 @@ func TestReconcileIngressUpdateReenqueueRollout(t *testing.T) {
 			ing = updated
 
 			// Advance the time by 5s.
-			fakeClock.SetTime(fakeClock.Now().Add(5 * time.Second))
+			const drift = 5
```

Comment:
Can this have a unit?

Reply:
The beauty of go numeric constants is that they are typeless — if I give it a type, in some calls I'll need to make casts. This way it's simpler.

Reply:
Change to driftSecs?